src/new/README.md

```diff
@@ -0,0 +1,60 @@
+# New Wallet Lib
+
+### Start wallet
+
+*Parameters:*
+
+- network: Network you want the wallet connected to. E.g., 'testnet' or 'mainnet'.
+- server: Server you want to connect to. E.g., 'https://node1.mainnet.hathor.network/v1a/'.
+- seed: 24 words of this wallet in a string separated by space.
+- tokenUid: Optional parameter if you want this wallet to work with an specific token only. E.g., '0000035f21f454686b3a01b59ae1345011501d0fcd8973d25d3d6ebe6f1c00bb'.
```

Comment:
I think it should be a map of `tokenSymbol` to `tokenUid`. So, methods can receive the `tokenSymbol` instead of the `tokenUid`. What do you think?

Reply:
I don't think we should do it because we could have in the network two different tokens with the same symbol, so the UID is the only unique data we have.

Reply:
As it is a wallet, we can go both ways because only registered tokens would be allowed, and we don't allow to have two registered tokens with the same symbol.

But I agree that it is safer to give the `tokenUid`.